src/components/card/CardSidebarTabDetails.vue

```diff
@@ -176,6 +177,48 @@ export default {
 				stringify: this.stringify,
 				parse: this.parse,
 			},
+			shortcuts: [
+				{
+					text: t('deck', 'Today'),
+					onClick() {
+						const date = new Date()
+						date.setDate(date.getDate())
+						date.setHours(23)
+						date.setMinutes(59)
+						return date
+					},
+				},
+				{
+					text: t('deck', 'Tomorrow'),
+					onClick() {
+						const date = new Date()
+						date.setDate(date.getDate() + 1)
+						date.setHours(23)
+						date.setMinutes(59)
```

Comment:
I think we should rather use the current time for the entries that are not due today.

Using 23:59 for today is also not ideal but I guess it will be hard to find a good other default time.

Reply:
cc @nextcloud/designers on this one. Any good idea if the user picks a day how we should determine the best time to set then for a due date reminder?

Reply:
@juliushaertl @jakobroehrl as usual: What do others like e.g. Trello do? :) One could argue for any of various solutions:
- Use 23:59 (end of day)
- Use the current time (or e.g. closest half hour to current time)
- Use 12:00 midday or some other default
- Remember whatever was used last

But it’s always best to check what another widely used solution does, because that’s what people are used to. :)

Reply:
Todo Tools like todoist or ticktick have All Day due dates. So I use 23:59 for today.
Trello uses the now due date. 
Meistertask uses 12:00 for the current date.

I would use 23:59. :)